api-admin/src/main/java/bio/terra/pearl/api/admin/service/DataRepoExportExtService.java

```diff
@@ -27,12 +29,12 @@ public DataRepoExportExtService(
     this.studyEnvironmentService = studyEnvironmentService;
   }
 
+  @EnforcePortalPermission(permission = "tdr_export")
```

Comment:
I really like the idea of using an annotation for this. It makes it super easy to spot which permissions are required, and as our roles and permissions become more widely used, it makes it much easier to spot when a permission check is missing.

Let's see how this one goes, but I do think it would be slick to have other auth annotations like `@EnforceStudyAuthed`